Package.swift

```diff
@@ -30,9 +30,9 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/IBM-Swift/LoggerAPI.git", .upToNextMajor(from: "1.0.0")),
-        .package(url: "https://github.com/apple/swift-nio.git", from: "1.0.0"),
-        .package(url: "https://github.com/apple/swift-nio-extras.git", from: "0.1.3"),
-        .package(url: "https://github.com/amzn/smoke-http.git", from: "1.0.0"),
```

Comment:
What is the reason for moving away from package versions here?

Reply:
SwiftNIO 2 is being actively developed and takes advantage of some optimizations in Swift 5 and later. We will have to move to it sooner or later.

Reply:
Oops, sorry. I meant to ask on the transition from 1.0.0 on smoke-http to master branch dependency

Reply:
Oh, I see. Yes, we don't want to do that long term but a SwiftNIO 2 based SmokeHTTP hasn't been tagged yet.

Reply:
Gotcha. Thanks for the clarification!